contracts/adapters/FeeSplitAdapter.sol

```diff
@@ -78,9 +79,9 @@ contract FeeSplitAdapter is BaseAdapter, TimeLockUpgrade {
      */
     function accrueFeesAndDistribute() public {
         streamingFeeModule.accrueFee(setToken);
-        
+
         uint256 totalFees = setToken.balanceOf(address(manager));
```

Comment:
This change is in a11eb74.

This approach requires that any time a fee extension contract is added or replaced the fee recipient is updated (by mutual upgrade) as well. 

Have added a `distribute` method to the extensions that lets distribution happen whether the extension is authorized or not. This might be necessary if `accrue` is called on the streaming fee module during a transition from one extension to another, or fees went undistributed for other reasons.